app/public/js/services/firebaseService.js

```diff
@@ -1,4 +1,14 @@
-angular.module('FirebaseService', []).service('firebaseService', function($rootScope) {
+angular.module('FirebaseService', []).service('firebaseService', function($firebaseArray) {
+
+  this.getAll = function(key) {
+    var ref = firebase.database().ref().child(key);
+    return $firebaseArray(ref);
+  }
+
+  this.getImageFullPath = function(key) {
+    var storageRef = firebase.storage().ref();
+    return storageRef.child(key).getDownloadURL();
```

Comment:
For consistency reasons, if you put child inside the ref variable, do the same thing here.